Reworked Holy Dagger Textures

KantenDestroyer opened the pull request:
resolves #5 
![Screenshot 2022-10-15 180243](https://user-images.githubusercontent.com/57726572/195996428-a18be0bc-831a-47c1-8c7e-535f8b436573.png)

Commit:
Initial commit

Commit:
Added Heart of Immortality as baseitem without functionality, aswell as Void Heart

Commit:
Added Heart of Immortality as baseitem without functionality, aswell as Void Heart

Commit:
Merge remote-tracking branch 'origin/master'

Commit:
Test

Hempflingclub committed:
New Branch to become to sole contributor instead misconfigured Git User

Hempflingclub committed:
Inital Commit

Hempflingclub committed:
Added run folder to .gitignore

Hempflingclub committed:
Delete run folder

Hempflingclub committed:
Test Commit

Hempflingclub committed:
Test Commit

Hempflingclub committed:
Added Item Tab for Immortality

Hempflingclub committed:
Added Test Functionality to Heart of Immortality
Added Full Functionality to Void Heart (refilling Food)
Added TickHandler
Made Items their own Classes
Added NbtData Saving

Hempflingclub committed:
Changed .gitignore

Hempflingclub committed:
Delete .idea directory

Hempflingclub committed:
Applied Immortality Functionality
You can level your Immortality for bonus Powers and to achieve a Void Heart
You gain an extra Heart for every Death
You cannot die
When falling out of world you go to your bed, otherwise your dimension spawn, otherwise overworld spawn
You won't gain an extra Heart for Void Damage (out of world or /kill)

Fixed Item Overlay Popup when consuming heart

Hempflingclub committed:
Added Effects for better Client visualisation of whether Immortal or VoidHeart is applied, also overworked all prior used of Effects

Hempflingclub committed:
Added Liver of Immortality
The Liver gives Immortality, but when dying you lose a heart instead of, leveling
When LiverImmortality reaches 0 Hearts, the killing Player will be said to have resulted in them losing their immortality
If the Immortal died themselves it will simply say, that they've lost their Immortality, instead of prior usage being giving immortality up

Added Holy Dagger
Holy Dagger can be used by Real Immortals (having used Heart of Immortality)
When used it will remove their Liver of Immortality and break, thus temporarily weakining them
After 5 Minutes the weakening will end, and they'll have regrown their Liver of Immortality

Fixed prior implementation of giving or taking extra Hearts
now will no Longer result in ServerSide Exceptions, whilst still resulting in their expected behavior

Hempflingclub committed:
Release 0.9
Added TODO list in [readme.md]

Hempflingclub committed:
Added things to TODO list

Hempflingclub committed:
Completed TODO list At Commit Time

Added Achievments for every significant Step (not for Leveling)
Added Item Descriptions
Added Crafting Recipes for Void Heart & Holy Dagger
Added New Secret Effect

Fixed Immortal Hearts staying behind
Fixed Immortals burning or without Air if dying whilst burning/drowning
Fixed Sounds only being on Immortal Player Client, now played for others aswell

Hempflingclub committed:
Pushed Translation which was forgotten before

Hempflingclub committed:
Added Dependency on Player Events API

Hempflingclub committed:
API and Events
Removed API as it only works ServerSide, thus disabling the mod from being used in SinglePlayer

Added Events standalone, without API, whilst allowing usage of them even in SinglePlayer
Failed to do ServerWorld Nbt Saver, added it to TODO

Hempflingclub committed:
Create gradle-publish.yml

Hempflingclub committed:
API and WorldDataSaving
Added Cardinal Components API as it "easily" solves the problem of attaching data to the world without it being discarded upon saving

This API will likely stay, as a manual implementation would not be worth it, especially as it's unlikely to port the Mod to versions not supported by the API

Hempflingclub committed:
Delete .github/workflows directory

Hempflingclub committed:
Reworked Base Functionality, Update to 0.9.9

When consuming Heart of Immortality / Void Heart or Liver of Immortality you no longer lose their effects

Consuming Liver of Immortality after losing Hearts, you regain your Original Hearts

You can use Holy Dagger 3 times in a second to lose all your Hearts, not to remove your Liver of Immortality

You will constantly be extinguished from now on, if Immortal, or False Immortal

Added Players having True Immortality radiating their Immortality, causing their Armor and held Items being slowly repaired

Fixed Bug of Player killing themselves on their last False Immortality Heart causing the Server to crash

KantenDestroyer committed:
Changed mod Icon because fun

KantenDestroyer committed:
Merge pull request #1 from Hempflingclub/master

Update to latest Commit

KantenDestroyer committed:
Changed mod icon to fulfil requested change by @Hempflingclub

Hempflingclub committed:
Merge pull request #2 from Hempflingclub/dev-Artists

Update mod icon

Hempflingclub committed:
Updated TODO list, planned until 16.10.2022

Hempflingclub committed:
Merge pull request #3 from Hempflingclub/master

Update to latest master Commit

Hempflingclub committed:
Renamed some things for Code Optimization

Hempflingclub committed:
Updated TODO List for 1.0 release

Hempflingclub committed:
Can't change Effect Names dynamically, so instead Command usage is planned
Initial Testing of Custom NameTag color

Hempflingclub committed:
Merge remote-tracking branch 'origin/dev-Artists'

Hempflingclub committed:
Updated TODO list, things for Devs

Hempflingclub committed:
Rebalancing and outsourcing too unreadable Methods

Changed Basic Death Leveling, now you don't gain (1) Heart per Death, but instead you gain (1) Armor Toughness, and every absorbed Liver now gives (1) Heart instead, whilst still temporarily taking (5) Hearts when taken out

Hempflingclub committed:
Removed completed Work from [readme.md]

Hempflingclub committed:
Commands | Armor/ArmorToughness
Added Command to get info about your current Status changes by Immortality

Added Armor gained on 4th and every 5th Death from then on
Fixed an output Text not being translatable

Hempflingclub committed:
Added working Particles to Immortality's Death prevention

Hempflingclub committed:
Fix Skin not hardening properly

Added Bugs to TODO of Devs to fix

Hempflingclub committed:
Added Particle Bug to TODO list of Devs

Hempflingclub committed:
Create gradle-publish.yml

Hempflingclub committed:
Update gradle-publish.yml

Hempflingclub committed:
Fixed Crash on consuming Liver of Immortality

Hempflingclub committed:
Merge remote-tracking branch 'origin/master'

Hempflingclub committed:
Update gradle-publish.yml

Hempflingclub committed:
Fixed unintended Liver of Immortality behaviour

Hempflingclub committed:
Merge remote-tracking branch 'origin/master'

Hempflingclub committed:
Update gradle-publish.yml

Hempflingclub committed:
Create gradle.yml

Hempflingclub committed:
gradlew executable

Hempflingclub committed:
Update readme.md

Hempflingclub committed:
Delete gradle-publish.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Update gradle.yml

Hempflingclub committed:
Fixed Liver of Immortality, Fixed Command output

Hempflingclub committed:
Merge remote-tracking branch 'origin/master'

Hempflingclub committed:
Fixed Particles for real now, the work

Fixed Bug where Player could kill themselves on Last Life again, and thus lose False Immortality without dying

Hempflingclub committed:
Fixed null bug of Player getting killed by other Player without, them registering them as Attackers, because of flick kills

Hempflingclub committed:
Update readme.md

Hempflingclub committed:
Changed Workload to current situation

Hempflingclub committed:
Merge remote-tracking branch 'origin/master'

KantenDestroyer committed:
Fixed error Message in console from advancement liver_of_immortality

I'm cool B)

Hempflingclub committed:
Merge pull request #4 from KantenDestroyer/master

Fixed error Message in console from advancement liver_of_immortality

Hempflingclub committed:
Added newly found Bugs and new Bonus Feature

KantenDestroyer committed:
Reworked Holy Dagger Textures

KantenDestroyer committed:
Merge branch 'Hempflingclub:master' into master

Hempflingclub reviewed (changes requested):
Worse than before, please rework again

Hempflingclub committed:
Fixed most Bugs that have been reported and listed in TODO

Hempflingclub committed:
Fixed Lost Data Bug through End Portal

Changed implementation to use Components now for PlayerData

Removed PlayerDataSaverMixin.java

Refactored to accompany new Component implementation

Refactored overusage of ImmortalityData.java into ImmortalityStatus.java

Implementated Syncing of Playerdata

Fixed Bug of Item Rendering happening when not originally intended

Hempflingclub committed:
Added Newly found Advancement Bug to Devs
Fixed Bug of Liver of Immortality not working as intended in worlds younger than 5 Minutes

KantenDestroyer committed:
Merge branch 'Hempflingclub:master' into master

Hempflingclub committed:
Fixed Bugs and added Achievment Double Hearted

Fixed Bug of Not being able to get Achievment for extracting Liver when already True Immortal

Fixed Bug of Command not displaying that you need to extract your liver when you already have 30 Deaths

KantenDestroyer committed:
Merge branch 'Hempflingclub:master' into master

KantenDestroyer committed:
Update gradle.yml

KantenDestroyer committed:
Update gradle.yml
